test_files/reexport.declaration/reexport_named.ts

```diff
@@ -0,0 +1,3 @@
+// reexport_named.ts exports symbols, but doesn't declare any of them, so its .d.ts
```

Comment:
"declare" is a bit overloaded here, is there a better word for it?  "declare them in local scope"?

Reply:
reworded